### DISCUSSION TITLE
ScrollArea and Table

### DISCUSSION THREAD

**Discussion by converter-user-old** [mantinedev/mantine#524] [Q&A] [answered]
How do i use a ScrollArea and a Table together?

So that when the Table is to big the ScrollArea will give me a scrollbar

**Comment by rtivital** [comment, marked as answer]
You just place Table inside ScrollArea, is there any problem with that?

**Reply by converter-user-old** [reply]
https://codesandbox.io/s/loving-darkness-uh8qm

That doesn't work :(

**Reply by rtivital** [reply]
You need to set ScrollArea height – https://codesandbox.io/s/nameless-https-sjyzn

**Reply by converter-user-old** [reply]
Okay but when i want the Table over the whole page? (For example in a AppShell)

**Reply by rtivital** [reply]
Set height with calc and vh, for example `calc(100vh - 60px)`

**Reply by converter-user-old** [reply]
Thanks that works 👍🏻
